data/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
+find_package(PythonInterp REQUIRED)
+
 set(OPENCC_DICT_BIN opencc_dict)
-set(DICT_MERGE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
-set(DICT_REVERSE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/reverse.py)
+set(DICT_MERGE_BIN ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
```

Comment:
Refer to usages of the `PYTHON_EXECUTABLE` variable in the CMake module
https://github.com/Kitware/CMake/blob/master/Modules/FindPythonInterp.cmake#L126
Double quotes around the variable were used.

Reply:
Agree with you. The ```PYTHON_EXECUTABLE``` should be quoted, code updated.